common/predictive-text/index.ts

```diff
@@ -64,7 +64,7 @@ namespace com.keyman.text.prediction {
      */
     constructor(capabilities: Capabilities, worker?: Worker) {
       // Either use the given worker, or instantiate the default worker.
-      this._worker = worker || new Worker(LMLayer.asBlobURI(LMLayerWorkerCode));
+      this._worker = worker || DefaultWorker.constructInstance();
```

Comment:
🤩 nice!

Reply:
Thanks for the conversations; they definitely made for a cleaner final product.  Occasionally I develop a bit too "agile" for my own good, but that is what code reviews are for, after all.